libs/model/src/main/kotlin/de/cyface/model/CalibrationJob.kt

```diff
@@ -44,12 +45,21 @@ class CalibrationJob : Job {
     /**
      * The number of locations which were filtered due to a rotated device.
      */
+    @Suppress("MemberVisibilityCanBePrivate") // Part of the API
     var rotatedLocations = 0
         private set
 
+    /**
+     * The number of locations which where filtered due to invalid speed, accuracy or time gaps.
+     */
+    @Suppress("MemberVisibilityCanBePrivate") // Part of the API
+    var invalidLocations = 0
+        private set
```

Comment:
To be able to show a warning in the client UI when all points of a measurement were filtered because of GNSS filters (speed, accuracy, gps time gaps like Huawei energy settings)